lessons/homework/ostrava3/tasks.yml

```diff
@@ -0,0 +1,139 @@
+
+tasks:
+
+- section:
+    markdown: |
+      V těchto projektech je určitě něco nového.
+      Nezadávají postup, jak na odpověď přijít, ale to určitě zvládneš!
+
+- markdown: |
+    Co vrací funkce `range`?
+
+- markdown: |
+    Co se stane, když pro příkaz `for`
+    zavoláš funkci `range` se dvěma parametry?
+
+- markdown: |
+    Se třemi?
+
+- section:
+    markdown: |
+      Želví kreslení! Tahle sekce tě naučí myslet jako programátorka počítačové grafiky.
+      Je dobré jednotlivé projekty dělat postupně, a když něco začne fungovat, znovu si
+      řešení projít a zamyslet se nad ním.
+
+- markdown: |
+    Nakresli trojúhelník.
+
+    <img src="static/03_trojuhelnik.png" alt="trojúhelník" style="max-width: 100%; max-height: 60px">
+
+    __Poznámka:__ Rovnostranný trojúhelník má vnitřní úhly 60°. Želva se ale otáčí o vedlejší úhel 180 - 60 = 120°.
+
+    <img src="static/03_diagram.png" alt="diagram" style="max-width: 100%; max-height: 80px">
+
+- markdown: |
+    Nakresli domeček!
+
+    <img src="static/04_domecek.png" alt="domeček" style="max-width: 100%; max-height: 60px">
+
+    _Jak pravil Pythagoras, délka šikmé čáry v domečku je √2-krát délka stěny. O funkci na odmocninu jsme mluvili na srazu._
+
+- markdown: |
+    Nakresli vesnici!
+
+    <img src="static/05_vesnice.png" alt="vesnice" style="max-width: 100%; max-height: 60px">
+
+- markdown: |
+    Nakresli pětiúhelník, šestiúhelník, sedmiúhelník, osmiúhelník.
+
+    <img src="static/06_uhelniky.png" alt="mnohoúhelníky" style="max-width: 100%; max-height: 60px">
+
+    _Vnitřní úhel pravidelného <var>n</var>-úhelníka má 180 × (1 - 2/<var>n</var>) stupňů._
+
+    _Aby byly tvary zhruba stejně veliké, použij pro <var>n</var>-úhelník délku strany např. 200/<var>n</var>_
+
+- markdown: |
+    Nakresli <var>n</var>-úhelník, kde <var>n</var> zadá uživatel.
+
+- markdown: |
+    Želva umí jen rovné čáry. Jde pomocí rovných čar nakreslit kolečko?
+
+    _Nevíš-li, přeskoč na další projekt a pak se sem vrať._
+
+- markdown: |
+    Nakresli pětadevadesátiúhelník.
+
+- markdown: |
+    Nakresli takovýto ornament:
+
+    <img src="static/10_ornament.png" alt="ornament" style="max-width: 100%; max-height: 80px">
+
+    _Nevíš-li si rady, vezmi pravítko a změř délky jednotlivých čar. Doporučuju začít od středu._
+
+- markdown: |
+    Nakresli takovýto ornament:
+
+    <img src="static/11_ornament.png" alt="ornament" style="max-width: 100%; max-height: 90px">
+
+- markdown: |
+    Nakresli spirálu.
+
+    <img src="static/12_spirala.png" alt="spirála" style="max-width: 100%; max-height: 140px">
+
+    _Od ornamentů ke spirále se dostaneš stejným způsobem jako od `n`-úhelníků ke kolečku._
+
+- markdown: |
+    Napiš obrázek! Třeba les ze stromů, město z domů, nebe z hvězd,
+    dav ze smajlíků, hradbu z věží, zikkurat ze schodů, ...
+
+    Můžeš použít i nějakou z funkcí modulu `random`.
+
+    Podle toho, jak jste se na sraze domluvili, pošli řešení e-mailem
```

Comment:
I'd remove this note as you did in the second handout. In Ostrava, girls are sending all home projects to the dropbox.